packages/storybook-react/src/stories/Paragraph.stories.tsx

```diff
@@ -1,22 +1,18 @@
 import readme from '@nl-design-system-unstable/documentation/componenten/_paragraph.md?raw';
 import { Meta, StoryObj } from '@storybook/react';
-import { Paragraph } from '@utrecht/component-library-react/dist/css-module/index';
+import { Paragraph } from '@utrecht/component-library-react';
 import tokensDefinition from '@utrecht/components/paragraph/tokens.json';
 import tokens from '@utrecht/design-tokens/dist/index.json';
-import React from 'react';
-import { designTokenStory } from './util';
 
-const meta = {
+export const Default: Meta<typeof Paragraph> = {
```

Comment:
Ik begreep uit deze recente blogpost van Storybook dat het beste zou zijn om `meta = { /* ... */ } satisfies Meta<typeof Paragraph>` te doen, als ik het goed onthouden heb.

Zie: [Improved type safety in Storybook 7](https://storybook.js.org/blog/improved-type-safety-in-storybook-7/)

Reply:
`export const Default` wordt vaak gebruikt voor de primaire story, niet voor de metadata. Waarom zou deze hernoemd moeten worden van `meta` naar `Default` wat jou betreft?